react/week1/todo-app/src/todo.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+
+const Todo = () => {
+  const todoDetails = [
+    {
+      id: 1,
+      description: "Get out of bed",
+      deadline: "Wed Sep 13 2017",
+    },
+    {
+      id: 2,
+      description: "Brush teeth",
+      deadline: "Thu Sep 14 2017",
+    },
+    {
+      id: 3,
+      description: "Eat breakfast",
+      deadline: " Fri Sep 15 2017",
+    },
+  ];
```

Comment:
A minor comment but for static constants/variables then it's a good idea to keep them outside the functions that renders as it won't change